src/mudlet.pro

```diff
@@ -242,7 +251,8 @@ SOURCES += \
     TVar.cpp \
     VarUnit.cpp \
     XMLexport.cpp \
-    XMLimport.cpp
+    XMLimport.cpp \
+    ircmessageformatter.cpp
```

Comment:
Can this one and the corresponding `.h` be positioned in the correct case-insensitive sort order position?